code/package.json

```diff
@@ -82,7 +82,7 @@
     "@types/babel__traverse@npm:*": "patch:@types/babel__traverse@npm%3A7.20.6#~/.yarn/patches/@types-babel__traverse-npm-7.20.6-fac4243243.patch",
     "@types/babel__traverse@npm:^7.18.0": "patch:@types/babel__traverse@npm%3A7.20.6#~/.yarn/patches/@types-babel__traverse-npm-7.20.6-fac4243243.patch",
     "@types/node": "^22.0.0",
-    "@vitest/expect@npm:2.0.5": "patch:@vitest/expect@npm%3A2.0.5#~/.yarn/patches/@vitest-expect-npm-2.0.5-8933466cce.patch",
+    "@vitest/expect@npm:2.1.3": "patch:@vitest/expect@npm%3A2.0.5#~/.yarn/patches/@vitest-expect-npm-2.0.5-8933466cce.patch",
```

Comment:
Notice that this still points to a patched v2.0.5, that's probably not what we want. Either we should remove the patch (if we can because of the upgrade), or re-apply the patch to v2.1.3

Reply:
Thanks for pointing that out, I had overlooked that about the patch 🤦‍♂️
As for updating the `vitest` related packages, as you pointed out separately, there was no need to update them, so we have reverted the commits that update them.

[97f93b7](https://github.com/storybookjs/storybook/pull/29407/commits/97f93b7d714ec32d42e21d3e5f5b6d6a64c9bf03)